lib/logsene-api.js

```diff
@@ -27,14 +27,26 @@ var request         = require('request'),
     stringify       = require('../lib/util').safeJsonStringify;
 
 
-var logseneEsHost = process.env.LOGSENE_ES_HOST,
-    logseneUri = process.env.LOGSENE_URI,
-    es;
+function getEsHost(region) {
+  if (region == "EU") {
+    return process.env.LOGSENE_ES_HOST_EU;
+  }
+  return process.env.LOGSENE_ES_HOST;
```

Comment:
Does this mean that if somebody makes a typo and write UE we'll silently fall back to US and the person may wonder why the login is failing?

Reply:
Technically not, because only "EU" or "US" can come out of the interactive shell after transforming the input through `getValidRegionString`.
But you are right that it is not very clear. I have centralized all the region-related code in one place now to make it more easy to understand.